service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/ManageOrdersController.java

```diff
@@ -164,6 +169,11 @@ public void handleSubmittedEvent(@RequestBody CallbackRequest callbackRequest) {
             "internal-change-manage-order",
             updates);
 
+        if (userService.isJudiciaryUser()) {
+            log.info("Creating ORDER_UPLOADED work allocation task for case: {}", caseData.getId());
```

Comment:
can probably remove this logging, we have logging inside the `createWorkAllocationTask` function

Reply:
Also we may want to defer it into an EventHandler similar to how notifications are dispatched